search_bytes/searching_aes_keys.md

```diff
@@ -1,8 +1,25 @@
 ## Searching for AES Keys
 
-Thanks to Victor Muñoz, radare2 now has support of the algorithm he developed, capable of finding expanded AES keys with `/Ca` command. It searches from current seek position up to the `search.distance` limit, or until end of file is reached. You can interrupt current search by pressing `Ctrl-C`. For example, to look for AES keys in physical memory of your system:
+Thanks to Victor Muñoz, radare2 now has support of the algorithm he developed, capable of finding **expanded AES** keys with `/Ca` command. It searches from current seek position up to the `search.distance` limit, or until end of file is reached. You can interrupt current search by pressing `Ctrl-C`. For example, to look for AES keys in physical memory of your system:
+
 ```
 $ sudo r2 /dev/mem
 [0x00000000]> /Ca
 0 AES keys found
 ```
+
+If you are simply looking for plaintext AES keys in your binary, radare2 will point to them as symbols so `/Ca` will not find the AES key, but `is~AES` should conveniently point to it instead:
```

Comment:
r2 doesnt point the aes keys by the symbols. if the file you are analyzing have a symbol and the programmer decided to  name it "AES", then you'll get a symbol like this. Otherwise that's not going to work. imho this can be rephrased in a better way